### PR TITLE
Fix printing

### PR DESCRIPTION
**Pull request by emord**
Merge with https://github.com/dimagi/Vellum/pull/507/files

cc: @czue

### CODE REVIEW

File: corehq/apps/app_manager/xform.py [hunk 1]
```diff
@@ -553,6 +554,11 @@ def media_references(self, form):
         return list(set([n.text for n in nodes]))
 
     @property
+    def text_references(self):
+        nodes = self.findall('{h}head/{odk}intent[@class="org.commcare.dalvik.action.PRINT"]/{f}extra[@key="cc:print_template_reference"]')
```
Comment (dimagimon) [inline corehq/apps/app_manager/xform.py]: E501 line too long (139 > 115 characters)

File: corehq/apps/hqmedia/models.py [hunk 2]
```diff
@@ -589,6 +589,9 @@ def _add_menu_media(item, **kwargs):
                     for video in parsed.video_references:
                         if video:
                             media.append(ApplicationMediaReference(video, media_class=CommCareVideo, **media_kwargs))
+                    for text in parsed.text_references:
+                        if text:
+                            media.append(ApplicationMediaReference(text, media_class=CommCareMultimedia, **media_kwargs))
```
Comment (dimagimon) [inline corehq/apps/hqmedia/models.py]: E501 line too long (121 > 115 characters)

**Comment by czue** [comment]
:+1: to go when https://github.com/dimagi/Vellum/pull/507/ is merged

**Comment by emord** [comment]
https://github.com/dimagi/Vellum/pull/507  was merged

**Pull request merged**